api/interservice/authz/v2/project.proto

```diff
@@ -91,6 +91,7 @@ message ApplyRulesStatusResp {
     float percentage_complete = 3;
     bool failed = 4;
     string failure_message = 5;
+    bool cancelled = 6;
```

Comment:
Hopefully this is the kind of nitpick that you'll appreciate-- but I'd prefer we *try* to be consistent with the spelling of _cancelled_ or _canceled_. Given that below we use one `l` on https://github.com/chef/automate/pull/1502/files#diff-ab846629da975cfcfdd9a22c5cbafa28R234, could we go with that?